### PR TITLE
Add modals

### PR DESCRIPTION
**Pull request by takanorip**
- Add ActionModal and MessageModal
- Stop using ComponentMeta, ComponentStoryObj
- fix GitHub Actions

### CODE REVIEW

File: src/components/MessageModal/MessageModal.module.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  max-width: 450px;
+  max-height: 90vh;
+  padding: 40px 16px 24px;
+  margin: 0 auto;
+  overflow-y: auto;
+  background: #fff;
+  border-radius: 12px;
+}
+
+.modalBody.fullscreen {
```
Comment (8845musign) [inline src/components/MessageModal/MessageModal.module.css]: Is the prerequisite `box-sizing: border-box` ?

スクリーンショット 2023-09-25 22 42 00

There is an overflow

File: src/components/ActionModal/ActionModal.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      leaveTo={styles.panelLeaveTo}
+    >
+      <Dialog static={isStatic} onClose={onClose} className={styles.modal}>
+        <Dialog.Overlay className={clsx(styles.overlay, styles[opacityClassName])} />
+        <div className={clsx(styles.modalBody, !header && styles.headerLess, fixedHeight && styles.fixedHeight)}>
+          {header && <Dialog.Title className={styles.header}>{header}</Dialog.Title>}
+          <div className={styles.contents}>{children}</div>
+          <div className={styles.buttonContainer}>
+            {onPrimaryAction && primaryActionLabel && (
+              <Button block onClick={onPrimaryAction} aria-label={primaryActionLabel} variant={primaryActionColor}>
```
Comment (8845musign) [inline src/components/ActionModal/ActionModal.tsx]: [question]

`aria-label` is required?
Always have a text node.

File: src/components/MessageModal/MessageModal.module.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 100vh;
+  max-width: 100vh;
+  height: 100vh;
+  max-height: 100vh;
+  padding: calc(env(safe-area-inset-top) + 40px) 32px calc(env(safe-area-inset-bottom) + 24px) 32px;
```
Comment (8845musign) [inline src/components/MessageModal/MessageModal.module.css]: `env()` 👀
I did not know! Nice.

File: src/components/ActionModal/ActionModal.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  primaryActionColor?: 'primary' | 'alert';
+  closeLabel?: string;
+  overlayOpacity?: Opacity;
+  fixedHeight?: boolean;
+  showClose?: boolean;
+  open?: boolean;
+  isStatic?: boolean;
+};
+
+type Props = PropsWithChildren<BaseProps> & AllOrNone<{ onSecondaryAction: () => void; secondaryActionLabel: string }>;
```
Comment (8845musign) [inline src/components/ActionModal/ActionModal.tsx:25]: Type awesome!

File: src/components/ActionModal/ActionModal.tsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+import { opacityToClassName } from '../../utils/style';
+import { AllOrNone } from '../../utils/types';
+
+type Opacity = 'normal' | 'darker';
+
+type BaseProps = {
+  onClose: () => void;
+  onPrimaryAction?: () => void;
+  header?: string;
+  primaryActionLabel?: string;
```
Comment (8845musign) [inline src/components/ActionModal/ActionModal.tsx:15]: [question]
For example, do you have a use case for only non-primary, secondary actions?
I was wondering why primary is not required.

Comment (takanorip) [reply src/components/ActionModal/ActionModal.tsx:15]: This is a specification for cases where there is an action in the modal content.
I really want to make it required, but I have no choice but to make it optional to cover existing use cases. 😢

File: src/components/MessageModal/MessageModal.module.css (hunk 1 above)
Comment (takanorip) [reply src/components/MessageModal/MessageModal.module.css]: Fixed!

File: src/components/ActionModal/ActionModal.tsx (hunk 2 above)
Comment (takanorip) [reply src/components/ActionModal/ActionModal.tsx]: Fixed!

File: src/components/MessageModal/MessageModal.module.css (hunk 1 above)
Comment (8845musign) [reply src/components/MessageModal/MessageModal.module.css]: Nice!

File: src/components/ActionModal/ActionModal.tsx (hunk 2 above)
Comment (8845musign) [reply src/components/ActionModal/ActionModal.tsx]: OK!

File: src/components/ActionModal/ActionModal.tsx (hunk 5 above)
Comment (8845musign) [reply src/components/ActionModal/ActionModal.tsx:15]: I understood!

**Review by 8845musign — APPROVED** [review]

**Pull request merged**